modules/bwa/mem/main.nf

```diff
@@ -29,7 +30,7 @@ process BWA_MEM {
         -t $task.cpus \\
         \$INDEX \\
         $reads \\
-        | samtools view $args2 -@ $task.cpus -bhS -o ${prefix}.bam -
+        | samtools $args2 --threads ${split_cpus} -o ${prefix}.bam -
```

Comment:
What's the need here to remove `view`? I don't understand the context. What's the difference between the #raredisease context and the #sarek context?

Reply:
I remove view because in sarek we do need sorted bams, I assume they need the same in raredisease

Reply:
I'm lost. Isn't that what the `SAMTOOLS_SORT` subworkflow is for?

Reply:
Yes, but I can do that here in the same process if I change this part.
Data footprint if greatly reduced this way, and it's also more time/cpu efficient

Reply:
Is there any reason not to have `sort` there instead of `view`?

Reply:
@drpatelh ?